### PR TITLE
[IMP] Allow to use database neutralization for odoo 16 and above

### PR DESCRIPTION
**Pull request by hugosantosred**
Adds a --neutralize option to click-odoo-restoredb to use database neutralization for odoo >= 16.0

Related with issue #135

### CODE REVIEW

**Review by danielduqma — APPROVED** [review]
LGTM!

**Review by xAdrianCif — APPROVED** [review]
Tested and tech review. Tanks!

**Review by sbidoul — COMMENTED** [review]
Thanks for the contrib. Can you add a test?

File: click_odoo_contrib/restoredb.py [hunk 1]
```diff
@@ -35,6 +35,8 @@ def _restore_from_folder(dbname, backup, copy=True, jobs=1):
         # if it's a copy of a database, force generation of a new dbuuid
         reset_config_parameters(dbname)
     with OdooEnvironment(dbname) as env:
+        if neutralize and odoo.release.version_info >= (16, 0):
+            odoo.modules.neutralize.neutralize_database(env.cr)
```
Comment (sbidoul) [inline click_odoo_contrib/restoredb.py:39]: This should fail if the neutralize option is not supported otherwise this gives a false sense of success.

Comment (hugosantosred) [reply click_odoo_contrib/restoredb.py:39]: You're right, I'll change it to fail in case it's not supported.

Comment (hugosantosred) [reply click_odoo_contrib/restoredb.py:39]: Done, I've added the check for the neutralize option and the odoo version in the main function. I've also added the tests

**Comment by bealdav** [comment]
Finally mergeable ?

**Comment by hugosantosred** [comment]
For me, yes, the code is complete with the tests included

**Review by sbidoul — APPROVED** [review]
Thanks!

**Comment by sbidoul** [comment]
@hugosantosred oh, can you also update the README, and add a `newsfragments/143.feature` with a short changelog entry?

**Comment by hugosantosred** [comment]
> @hugosantosred oh, can you also update the README, and add a `newsfragments/143.feature` with a short changelog entry?

@sbidoul Added in https://github.com/acsone/click-odoo-contrib/pull/147, sorry for the delay

**Pull request merged**